### PR TITLE
Add support for RaggedTensors to mean_absolute_percentage_error.

### PR DESCRIPTION
**Pull request by pedro-r-marques**
Follow up to #46875.

### CODE REVIEW

File: tensorflow/python/keras/losses.py [hunk 1]
```diff
@@ -1328,6 +1328,15 @@ def mean_absolute_percentage_error(y_true, y_pred):
   return 100. * K.mean(diff, axis=-1)
 
 
+@dispatch.dispatch_for_types(
+    mean_absolute_percentage_error, ragged_tensor.RaggedTensor)
+def _ragged_tensor_mape(y_true, y_pred):
```
Comment (rchao) [inline tensorflow/python/keras/losses.py:1333]: Thanks for the PR! Can you provide more information where this is used?

Comment (rchao) [reply tensorflow/python/keras/losses.py:1333]: Sorry - I probably misunderstood. @tomerk do you have some context to review this?

Comment (pedro-r-marques) [reply tensorflow/python/keras/losses.py:1333]: There is the generic goal of making RaggedTensor support in keras work across all operations. There is a specific issue for loss functions  (#45403); PR #46875, introduces a solution for mean_squared_error; it would seem to me that all library supported loss functions should be modified to support RaggedTensors; I believe users would expect a consistent behaviour.

Comment (pedro-r-marques) [reply tensorflow/python/keras/losses.py:1333]: Also, please see #46875 and #46876. It would probably be good to assign them to @tomerk also if possible.

**Review by tomerk — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Review by tomerk — APPROVED** [review]
